conditional-kotlin/src/main/kotlin/com/linecorp/conditional/kotlin/ConditionalExtension.kt

```diff
@@ -16,11 +16,104 @@
 
 package com.linecorp.conditional.kotlin
 
-import com.linecorp.conditional.ComposedCondition
-import com.linecorp.conditional.Condition
+import com.linecorp.conditional.*
+import java.util.concurrent.TimeUnit
 
-infix fun Condition.and(condition: Condition): ComposedCondition = this.and(condition)
+/**
+ * Returns a newly created [ComposedCondition].
+ * This [ComposedCondition] is composed with the AND operator.
+ *
+ * @param condition the [Condition] to compose.
+ *
+ * @throws NullPointerException if the [condition] is null.
+ */
+infix fun Condition.and(condition: Condition): ComposedCondition = and(condition)
```

Comment:
It's a very small part, but I was curious about the infix function coding style.

The previously written code style exposed the `this`.
```kotlin
infix fun Condition.and(condition: Condition): ComposedCondition = this.and(condition)
```
But the current version seems to have removed `this`.
```kotlin
infix fun Condition.and(condition: Condition): ComposedCondition = and(condition)
```

In this regard, I searched for a convention suggested by the [Kotlin Style Guide](https://kotlinlang.org/docs/coding-conventions.html), but it seems that there is no related content. So, I have [raised an issue on the Kotlin Style Guide](https://github.com/Kotlin/kotlin-style-guide/issues/47).

Personally, I think that using the `this` is similar to using the original function to be made into an infix function, and it has the advantage of helping to understand the code.

However, since the original function is not complicated and there will be no major problems even if this is omitted, I think it is fine as it is now 🙂

Reply:
@doljae, I think you're right. I think it will be more helpful to understand this code when explicitly using `this` keyword. I will fix this. 👍

Reply:
Thank you for your review. I've fixed this on https://github.com/line/conditional/pull/7/commits/1c42ae1b7a196f722c3b9ce909a92f930e631941